components/work_templates/components/menu.tsx

```diff
@@ -95,7 +70,7 @@ const Menu = (props: MenuProps) => {
         props.onTemplateSelected(template, false);
     };
 
-    if (categories.length === 0) {
+    if (props.categories.length === 0) {
```

Comment:
nit: use (!props.categories.length)
nit: destructure at the beggining of the file (`const {x} = props;`) so your code looks better to avoid calling everything with `props.`